test/configSpec.js

```diff
@@ -1,71 +1,292 @@
 'use strict'
 
+const _ = require('lodash')
 const chai = require('chai')
 const expect = chai.expect
-const Config = require('../lib/config')
+const config = require('../lib/config')
 
-describe('Config', function () {
-  describe('constructor', function () {
-    it('should correctly convert the prefix', function () {
-      const config = new Config('a-long-PrEFiX')
+let originalEnv
+describe('config', () => {
+  beforeEach(() => {
+    originalEnv = _.cloneDeep(process.env)
+  })
 
-      expect(config.uppercasePrefix).to.equal('A_LONG_PREFIX_')
-    })
+  afterEach(() => {
```

Comment:
You can get rid of this if you just do `const originalEnv = _.cloneDeep(process.env)` at the top and `process.env = _.cloneDeep(originalEnv)` in `beforeEach`